### PR TITLE
Updating docstrings

### PR DESCRIPTION
**Pull request by victor-torres**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,3 @@
-===========
```
Comment (kmike) [inline README.rst:1]: This line gave a different header level; after this change scrapy-poet is on a same level as e.g. Usage.

If that's a problem for docs, I think a little of duplication is fine (i.e. pull in changelog, but not readme).

File: scrapy_poet/utils.py [hunk 2]
```diff
@@ -22,8 +22,50 @@ def get_callback(request, spider):
...
+                self.response = response
+
+            def __call__(self):
+                return ResponseData(
+                    url=self.response.url,
+                    html=self.get_data()
+                )
+
+            def get_data(self):
+                # make an api call
```
Comment (kmike) [inline scrapy_poet/utils.py:61]: to make an API call you would probably want an async def function

File: README.rst (hunk 1 above)
Comment (victor-torres) [reply README.rst]: reverted

File: scrapy_poet/utils.py (hunk 2 above)
Comment (victor-torres) [reply scrapy_poet/utils.py:62]: I've updated this example to use async/await.

**Comment by kmike** [comment]
Thanks @victor-torres!

**Pull request merged**